components/RatingsTable/index.tsx

```diff
@@ -21,24 +21,60 @@ interface ColumnData {
 interface Props {
   data: Array<PlayerRatings | GoalieRatings>;
   columnData: Array<ColumnData>;
+  pagination?: boolean;
+  teamPage?: boolean;
   // isLoading: boolean;
 }
 
 function RatingsTable({
   data: players,
   columnData,
+  pagination = false,
+  teamPage = false,
 }: // isLoading
 Props): JSX.Element {
   // ! add loading state
   const data = useMemo(() => players, [players]);
 
   // ! add loading state
-  const columns = useMemo(() => columnData, []);
+  const columns = useMemo(() => {
```

Comment:
Adds teamPage support (removes the team from the ratings table when it is on the team page).